src/applications/vaos/components/AppointmentTime.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { formatInTimeZone } from 'date-fns-tz';
+import { stripDST } from '../utils/timezone';
+
+/**
+ * Component for displaying appointment time in a consistent format with timezone
+ */
+export default function AppointmentTime({ date, timezone }) {
+  const formattedTime = stripDST(
+    formatInTimeZone(new Date(date), timezone, 'h:mm aaaa'),
+  );
+  const timezoneAbbreviation = stripDST(
+    formatInTimeZone(new Date(date), timezone, 'zzz'),
```

Comment:
Using zzz can potentially return a GMT prefixed timezone for Pacific/* and Asia/* locations and we have a mapping you can use [here](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/vaos/utils/timezone.js#L17-L20). Better yet, we can extract that portion of the logic from the `getUserTimezoneAbbr` function call to be re-used here.